### ISSUE TITLE
welt.de: ads

### ISSUE THREAD

**Issue by Symbai** [uBlockOrigin/uAssets#15918]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625)
- [X] I verified the issue is not caused by the browser built-in blocker, or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through VPN)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.welt.de/politik/deutschland/video242493243/Klimakleber-selbst-entfernen-Diese-Rechte-haben-Buerger-Jura-Professor-erklaert.html?source=puerto-reco-2_ABC-V16.1.B_evergreen`

### Category

ads

### Describe the issue

If you play the video it plays a video ad first. For non-German users you'll see its an ad if there is a count down at the bottom of the video, see screenshot. I've searched for similar issues and found this one #15698, tried the mentioned solution but it doesn't work or they already fixed it.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://user-images.githubusercontent.com/14368203/205734067-77f83ff7-0c81-4bda-9a32-41289feeebb8.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.45.2
Chromium: 107
filterset (summary): 
  network: 97974
  cosmetic: 135608
  scriptlet: 21762
  html: 0
listset (total-discarded, last-updated): 
  added: 
    adguard-social: 19368-24, 1d.6h.6m
    fanboy-annoyance: 83450-1363, 5d.4h.35m
    fanboy-cookiemonster: 43649-43649, 5d.4h.33m
    fanboy-thirdparty_social: 69-21, 5d.4h.34m
    ublock-annoyances: 4798-66, 1d.14h.11m
  default: 
    user-filters: 14-0, never
    DEU-0: 6051-0, 6h.37m
    easylist: 64993-59, 5d.8h.38m
    easyprivacy: 30936-1183, 18h.38m
    plowe-0: 3679-901, 3d.12h.24m
    ublock-abuse: 77-0, 4d.12h.30m
    ublock-badware: 4045-110, 1d.6h.7m
    ublock-filters: 31596-175, 1d.14h.12m
    ublock-privacy: 260-9, 4d.12h.31m
    ublock-quick-fixes: 389-18, 6h.36m
    ublock-unbreak: 1912-49, 2h.34m
    urlhaus-1: 7849-0, 6h.35m
filterset (user): [array of 14 redacted]
trustedset: 
  added: [array of 35 redacted]
  removed: 
    edge-scheme
hostRuleset: 
  added: [array of 1 redacted]
modifiedUserSettings: 
  advancedUserEnabled: true
  webrtcIPAddressHidden: true
modifiedHiddenSettings: 
  userResourcesLocation: [redacted]
supportStats: 
  allReadyAfter: 780 ms (selfie)
  maxAssetCacheWait: 419 ms
popupPanel: 
  blocked: 12
  blockedDetails: 
    welt.de: 3
    asadcdn.com: 2
    googletagmanager.com: 1
    kameleoon.eu: 3
    nmrodam.com: 2
    rapid7.com: 1
```
</details>

**Comment by JobcenterTycoon** [comment]
@mapx- the fix doesn’t work the real video is broken with this

**Comment by mapx-** [comment]
I tested in chrome

**Comment by JobcenterTycoon** [comment]
It also not working in chrome

**Comment by mapx-** [comment]
working in FF too on my end, I just have to click twice the play button

**Comment by JobcenterTycoon** [comment]
Do you get the blocked media in the logger?

**Comment by mapx-** [comment]
sure

**Comment by mapx-** [comment]
![image](https://user-images.githubusercontent.com/10303732/205740959-9154183f-8afe-41ed-baf5-61e462aa23e8.png)

**Comment by JobcenterTycoon** [comment]
Hmm for the the real video is broken and just show a single frame. I tried different other filter but none working

@okiehsch

**Comment by mapx-** [comment]
wait ... it seems it's only audio not the video too, is it what you get ?

**Comment by okiehsch** [comment]
Yes, the video is broken only audio is playing.

**Comment by mapx-** [comment]
@okiehsch @JobcenterTycoon 

could you test

```
@@||adnxs.com^$domain=welt.de
||adnxs.com/$xhr,domain=welt.de,removeparam,important
```

**Comment by okiehsch** [comment]
Works on my end.

**Comment by mapx-** [comment]
I get ads only if I disable those filters

**Comment by Symbai** [comment]
FYI: I tried the new filter above in two different Edge (chromium) versions, in one of it I still got ads. I tried to figure out whats the difference and it turned out that I still see video ads when [tracking protection](https://learn.microsoft.com/en-us/microsoft-edge/web-platform/tracking-prevention) in Edge setting is set to "strict". I can instantly reproduce this by switching from "strong" to "balance" (no ads) and back (ads). Also when its set to "strict" I have to click the play button twice. Maybe @JobcenterTycoon can see if that is the same on his/her end?

**Comment by JobcenterTycoon** [comment]
After i tested i get ads on firefox but not on chrome

**Comment by okiehsch** [comment]
>  i get ads on firefox

Do you have the tracking protetion enabled?
I can't reproduce.

**Comment by JobcenterTycoon** [comment]
I use the firefox developer edition with default settings. When i open the link directly and playing the video i get no ads but when i go to the startpage and play another video i get ads.

**Comment by okiehsch** [comment]
WithF irefox stable all videos are playing without issue on my end.

**Comment by okiehsch** [comment]
> What about to change the unused safari issue label to firefox issue ?

Done.

**Comment by mapx-** [comment]
No ad in FF too.

**Comment by Symbai** [comment]
@JobcenterTycoon @mapx- @okiehsch 

They fixed it already. I got video ads again (Edge Chromium, in-browser tracking protection set to lowest level), URL:

`https://www.welt.de/politik/deutschland/video242648901/Jetzt-live-im-TV-Aquadom-in-Berlin-geplatzt-eine-Million-Liter-Wasser-laufen-aus.html`

<details> 
  <summary>Screenshot </summary>

![image](https://user-images.githubusercontent.com/14368203/208069277-b04839a8-9a93-4a44-a09d-ddfcbcf08b80.png)
</details>

I don't want to open a new issue, can someone please re-open this one for me?

**Comment by mapx-** [comment]
Test

```
welt.de#@#+js(nostif, .call(null), 10)
*$frame,domain=welt.de,redirect-rule=noopframe
*$script,domain=welt.de,redirect-rule=noopjs
```

**Comment by JobcenterTycoon** [comment]
doesn’t work for me, also `welt.de#@#+js(nostif, .call(null), 10)` will bring the ad images back

**Comment by mapx-** [comment]
No video ads, no other ads FF / chrome on my side

**Comment by JobcenterTycoon** [comment]
The ad reinsertion images are on regular articles like `https://www.welt.de/vermischtes/article242714393/Sea-Life-Berlin-Grossaquarium-Aquadom-geplatzt-Zwei-Verletzte-Ueberall-liegen-tote-Fische-herum.html`

**Comment by mapx-** [comment]
right, the ad images are back on the home page at least

**Comment by mapx-** [comment]
test

```
@@*$image,domain=welt.de,3p
welt.de#@#+js(nostif, .call(null), 10)
*$frame,domain=welt.de,redirect-rule=noopframe
*$script,domain=welt.de,redirect-rule=noopjs
```

**Comment by JobcenterTycoon** [comment]
Seems to be they have issues because i get the damoh ads even with disabled adblocker. Crazy i see all ad requests with iframes and all (no content blocker, build in tracking protection or adblock DNS active) but get no banner ads 🤔

**Comment by mapx-** [comment]
no such issue on my end, did you test in chrome too ?

**Comment by mapx-** [comment]
I can replace the image exception by aost filter

```
welt.de#@#+js(nostif, .call(null), 10)
*$frame,domain=welt.de,redirect-rule=noopframe
*$script,domain=welt.de,redirect-rule=noopjs
welt.de##+js(aost, Object, /gdpr.*inlineScript/, 2)
```

**Comment by JobcenterTycoon** [comment]
Same result on chrome for me

@okiehsch

**Comment by JobcenterTycoon** [comment]
Update: works fine now on chrome

**Comment by Symbai** [comment]
Video ads are back: `https://www.welt.de/politik/deutschland/video243436099/Zivilcourage-nach-Messerattacke-Als-der-Taeter-ins-naechste-Abteil-ging-flogen-die-Koffer.html`

MS Edge browser inbuilt tracking protection is disabled.

**Comment by JobcenterTycoon** [comment]
`@@||eu-api-visit.kameleoon.eu/events?siteCode=$xhr,domain=welt.de` works for me on chrome @mapx-

**Comment by mapx-** [comment]
it seems ok for me too

**Comment by Symbai** [comment]
Not sure what I'm doing wrong but I still get video ads even with @JobcenterTycoon new filter. Did they fixed it already or is this depending on user location?

`https://www.welt.de/politik/deutschland/article243502795/Bundesfinanzhof-weist-Klage-gegen-Solidaritaetszuschlag-ab.html`

<details> 
  <summary> Screenshots</summary>

   ![image](https://user-images.githubusercontent.com/14368203/215459490-b9420699-89fa-45da-bf1e-9ff182155ead.png)

</details>

**Comment by JobcenterTycoon** [comment]
They are 👀 us. I can’t find a fix at the moment

**Comment by garry-ut99** [comment]
> Symbai : but I still get video ads 

Me too.

**Comment by Symbai** [comment]
In the comments it shows ads now. I saw similar ads before elsewhere on the site and they disappeared after some hours.

### URL address of the web page

`https://www.welt.de/politik/ausland/article254653234/Erdogan-wuenscht-sich-in-Telefonat-mit-Putin-intensiveren-Handel-mit-Russland.html`

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>




![image](https://github.com/user-attachments/assets/718753e7-5cdc-4215-a5e5-1eac15a76fa3)



</details>


### Configuration

<details>

```yaml
uBlock Origin: 1.61.0
Chromium: 130
filterset (summary):
 network: 120946
 cosmetic: 56969
 scriptlet: 27189
 html: 0
listset (total-discarded, last-updated):
 removed:
  urlhaus-1: null
 added:
  ublock-annoyances: 5919-12, 10m Δ
 default:
  user-filters: 21-0, never
  DEU-0: 7804-1, 3m
  easylist: 79217-616, 10m Δ
  easyprivacy: 53280-664, 10m Δ
  plowe-0: 3530-0, 3m
  ublock-badware: 12005-1, 10m Δ
  ublock-filters: 40686-157, 10m Δ
  ublock-privacy: 1543-22, 10m Δ
  ublock-quick-fixes: 223-0, 10m Δ
  ublock-unbreak: 2501-3, 10m Δ
filterset (user): [array of 21 redacted]
trustedset:
 added: [array of 43 redacted]
switchRuleset:
 added: [array of 1 redacted]
hostRuleset:
 added: [array of 1 redacted]
userSettings:
 advancedUserEnabled: true
 webrtcIPAddressHidden: true
hiddenSettings:
 userResourcesLocation: [redacted]
supportStats:
 allReadyAfter: 2333 ms (selfie)
 maxAssetCacheWait: 301 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 64
 network:
  welt.de: 5
  asadcdn.com: 1
  bing.com: 1
  chartbeat.com: 1
  cxense.com: 4
  doubleclick.net: 1
  facebook.net: 1
  googletagmanager.com: 1
  kameleoon.eu: 1
  kameleoon.io: 5
  nmrodam.com: 1
  ocdn.eu: 2
  onet.pl: 6
  outbrain.com: 32
  outbrainimg.com: 2
 extended:
  ##.c-ad-container
  ##+js(nostif, .call(null), 10)
  ##+js(rmnt, script, DisplayAcceptableAdIfAdblocked)
  ##+js(ra, onmousedown, .ob-dynamic-rec-link, stay)
```
</details>

**Comment by JobcenterTycoon** [comment]
@khrin i wonder for what these filters are
```
@@||asadcdn.com/adlib/$domain=petbook.de|welt.de
@@||ib.adnxs.com/ut/v3$domain=petbook.de|welt.de
```

they enabling video ads.

**Comment by Symbai** [comment]
Today there are unblocked ads. 

### URL address of the web page
`https://www.welt.de/`

`https://www.welt.de/politik/deutschland/article255808150/Koalitionsverhandlungen-Trotz-Sparansage-von-Merz-Koalition-plant-angeblich-Mehrausgaben-von-350-Milliarden-Euro-Liveticker.html`

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>




![Image](https://github.com/user-attachments/assets/95a6d214-ef7c-49a3-8c6f-3079708b3ade)

![Image](https://github.com/user-attachments/assets/d80e369e-81ec-4b8b-a1ef-f763a58966d2)

![Image](https://github.com/user-attachments/assets/c6e98f40-cc64-40e9-8ba4-8a54fc7921d1)



</details>

**Comment by stephenhawk8054** [comment]
Looks like a bug when moving the filters to `filters-general.txt` list. I have moved back the filters to `filters.txt`: https://github.com/uBlockOrigin/uAssets/commit/2a8c9d64c1b5e0c694cb23bf30d156a6836a290e .

**Comment by stephenhawk8054** [comment]
Yeah, it seems like a bug of the new list. I have added a generic one temporarily until the diff-update is populated in next update cycle.

**Comment by stephenhawk8054** [comment]
Yeah, that's what I don't understand as well and I overlooked that when I tested the new list: https://github.com/uBlockOrigin/uAssets/commit/bd0b01ef49dbc9f16b010fcb229fdbfbde067bbd

I just checked that the filters are in the list but didn't check whether they work or not before moving the real filters to the general file.

**Comment by mapx-** [comment]
could be you missed (while copying the filters) some if / endif ?

**Comment by stephenhawk8054** [comment]
I did check again many times, but I don't find the missed `if`.

The thing is for the filters before `! *** filters/filters-general.txt ***` or after `! *** filters/ubo-link-shorteners.txt ***`, they all work normally. Only the filters between that 2 lines don't work.

**Comment by gwarser** [comment]
It's a recurring issue with new line characters. It must be `\n` but often online editors insert only `\r`. Happened a few times already in "yearly" `filters-XXXX.txt`. Hopefully will be fixed this time.

**Comment by Symbai** [comment]
Now the live video stream is broken. The video is blurred, audio works fine. Disabling ublockorigin fixes it. I believe the video ad filters somehow breaks it. `https://www.welt.de/` 

<details>
<summary>Screenshot(s)</summary>

<img width="1285" height="546" alt="Image" src="https://github.com/user-attachments/assets/967597af-9559-47a8-84c4-ec000aee69fa" />


</details>

Edit: Not only live stream but all videos are effected: `https://www.welt.de/sport/video6883886d0e680a76f4e958ac/hulk-hogan-ist-tot-der-schwerste-skandal-war-ein-sextape-das-als-folge-einen-rassismus-skandal-produziert-hat.html`

**Comment by okiehsch** [comment]
Caused by
`welt.de##+js(aost, XMLHttpRequest.prototype.open, svonm.com)`
Disabling it will fix the videos but the ads will be back too.

**Comment by Symbai** [comment]
Looks like they're fighting back. This time its the other way around. You get the video but no audio (on homepage). If you try to play a recorded video (from the link above) it stays on pause. You can skip through it but it won't play.

**Comment by Symbai** [comment]
Ads appear on the pages of articles after a slight delay, but it is difficult to reproduce this. Sometimes it takes a while, sometimes you have to scroll up or down. Sometimes advertisements only appear on one page, sometimes not at all. I feel like it's more consistent to go to the main page and open an article rather than opening an article directly via the URL, but I could be wrong. Interestingly, the advertisements **can** disappear again after 10-15 seconds. 

`https://www.welt.de/politik/ausland/article69673887733b8ea321018356/ukraine-krieg-manche-europaeischen-staatschefs-sind-vielleicht-aus-europa-aber-nicht-fuer-europa-sagt-selenskj-in-davos-liveticker.html` 

<details>
<summary>Screenshot(s)</summary>

<img width="639" height="355" alt="Image" src="https://github.com/user-attachments/assets/693018f5-83ec-4a4f-9329-e8134f44c1cd" />


</details>

**Comment by JobcenterTycoon** [comment]
Can’t reproduce at the moment

**Comment by Symbai** [comment]
@JobcenterTycoon Can you give it another try please? I'm not getting the ads on every article but I'm getting them constantly now. Url of the ads always start with `https://9.html-load.com/player/www.welt.de/`

Edit: Now its even on both sides:

<details>
<summary>Screenshot(s)</summary>

<img width="945" height="464" alt="Image" src="https://github.com/user-attachments/assets/faae1bdf-1561-4d77-8914-cc39881caf63" />

`https://www.welt.de/politik/deutschland/article69786a00aa4bbccad13d2de4/duisburg-dortmund-hagen-gezieltes-einwandern-in-sozialsysteme-nrw-metropolen-kuerzen-1181-mal-buergergeld-fuer-migranten.html`


</details>

**Comment by Symbai** [comment]
Im still getting these ads on left or right side. I've removed any browser ad protection. I've ran a private window with just ublock enabled to see if it might be another extension causing it but no luck with that.

<details>
<summary>Screenshot(s)</summary>

<img width="3772" height="1941" alt="Image" src="https://github.com/user-attachments/assets/5a32bdda-a1a3-4a35-80e6-a93612de943a" />

`https://www.welt.de/vermischtes/article69fe5a91cd4a28c4700216d6/berlin-charlottenburg-tat-steckt-uns-in-den-knochen-fdp-politiker-und-partner-bei-homophobem-angriff-verletzt.html#Comments`

These are my filters:

<img width="791" height="1801" alt="Image" src="https://github.com/user-attachments/assets/ced106cc-da1d-4890-9bc6-2aecda8658fe" />

</details>

**Comment by stephenhawk8054** [comment]
Please always share your troubleshooting information, because it always includes which browser you are using. Don't crop your screenshots.
